grpc-1.66.yaml

```diff
@@ -96,22 +99,26 @@ pipeline:
       sh -c 'rm $0 && ln -s /etc/ssl/certs/ca-certificates.crt $0' "{}" \;
 
 subpackages:
-  - name: py3-grpcio
+  - name: ${{package.name}}-py3-grpcio
```

Comment:
I am not completely sure about this package name probably we can use py-3-grpcio-${{vars.major-minor}}
instead of grpc-1.66-py3-grpcio

Reply:
Can we update the checklist with more detailed naming styling? 🙏
I originally did something like that, but ended up not doing it because of the checklist recommendation.

For `major-minor` - is there a built-in variable I can use, or do I need to do the var transformation myself?

Reply:
You need to use var transformation

https://github.com/wolfi-dev/os/blob/main/gitlab-kas-17.4.yaml#L12-L15

Reply:
For regular sub-package ${{package.name}}-xyz would work , but this seems like a py3-package  , would defer to @EyeCantCU or @pnasrat for the naming styling here

Reply:
I think py3-grcpio-${{vars.major-minor}} for the versioned one makes sense. We wouldn't expect the two versions of the python module to be co-installable

The checklist is somewhat simplified and predates version streaming I believe - I'm not sure who owns that but there are multiple cases we do other things

Reply:
In agreement with Pris